utils/calendrical_calculations/src/rata_die.rs

```diff
@@ -67,14 +67,25 @@ impl RataDie {
         self.0 as f64
     }
 
+    /// Subtracts a number of days from this `RataDie` in a const-friendly way
+    pub const fn sub(self, rhs: i64) -> RataDie {
+        let result = Self(self.0 - rhs);
+        #[cfg(debug_assertions)]
+        result.check();
+        result
+    }
+
     /// Calculate the number of days between two `RataDie` in a const-friendly way
-    pub const fn const_diff(self, rhs: Self) -> i64 {
+    pub const fn diff(self, rhs: Self) -> i64 {
         self.0 - rhs.0
     }
```

Comment:
Should we discuss the names of these functions? I had suggested `sub_i64` and `sub_rata_die` in the other thread. Maybe `sub_i64` and `sub`?

Reply:
Temporal has `difference` as a static method

I won't die on this hill

Reply:
I think `Sub<Self>` is the default sub trait so should be the one called `sub` here.

Removing one of these is also fine.